frontendUi/build.gradle

```diff
@@ -19,7 +19,7 @@ intellij {
 patchPluginXml {
   // Note: update scripts/enforce-binary-compatibility when updating the compatibility data
   sinceBuild '192.*' // aka 2019.2
-  untilBuild '201.*' // aka 2020.1 (exclusive)
+  untilBuild '201.6668.122' // (exclusive)
```

Comment:
Hmmm shouldn't it rather be `202.*` (as in, support all 2020.1.* versions but not yet 2020.2)? Pls note that the range is **exclusive** (or do we have a reason to support builds up to 201.6668.121 inclusively but not 201.6668.122?)